src/reporters/console/console-reporter.js

```diff
@@ -154,10 +154,10 @@ export default class ConsoleReporter extends BaseReporter {
     this.stopProgress();
 
     const totalTime = (this.getTotalTime() / 1000).toFixed(2);
-    let msg = `Done in ${totalTime}s.`;
+    let msg = `Done in ${totalTime} s.`;
     if (showPeakMemory) {
       const peakMemory = (this.peakMemory / 1024 / 1024).toFixed(2);
-      msg += ` Peak memory usage ${peakMemory}MB.`;
+      msg += ` Peak memory usage ${peakMemory} MB.`;
```

Comment:
Shouldn't that be `MiB` instead of `MB` if we want to comply to SI? (Refs: https://en.wikipedia.org/wiki/Mebibyte)

Reply:
OK, done in the second commit, thanks.